docs/data_usage_guides/affected_people.md

```diff
@@ -2,29 +2,114 @@
 
 ---
 
+## Internally-Displaced Persons <a id=”idps”></a>
+
+The data for this sub-category is taken from the
+[International Organization for Migration](https://www.iom.int/) (IOM)'s
+[displacement tracking matrix](https://dtm.iom.int/) (DTM),
+which collects and analyzes data on population movements,
+displacements, and mobility patterns to provide timely information
+for humanitarian response efforts.
+THe DTM data used in HAPI is taken from their
+[publicly accessible API](https://dtm.iom.int/data-and-analysis/dtm-api),
+however we note that there are many country-specific DTM datasets
+[available on HDX](https://data.humdata.org/dataset/?dataseries_name=IOM%20-%20DTM%20Baseline%20Assessment&dataseries_name=IOM%20-%20DTM%20Event%20and%20Flow%20Tracking&dataseries_name=IOM%20-%20DTM%20Site%20and%20Location%20Assessment&organization=international-organization-for-migration&q=&sort=last_modified%20desc&ext_page_size=25),
+which contain more detail and disaggregation.
+
+### Summary
+
+{{ read_yaml('data_usage_guides/subcategory_details/idps_details.yaml') }}
+
+### Parameters Returned
+
+The table below describes the parameters returned from this endpoint.
+For available query parameters, please see the
+[API sandbox](https://hapi.humdata.org/docs#/Affected%20people/get_idps_api_v1_affected_people_idps_get).
+
+{{ read_yaml('data_usage_guides/endpoint_parameters/idps_parameters.yaml') }}
+
+### Transformations applied
+
+None
+
+### Usage notes
+
+* To obtain the number of IDPs per year in a given location,
+  only the most recent reporting round per operation should be aggregated
```

Comment:
I'm wondering whether this would be clear to the user, I think it is but I'm unfamiliar with the data